tera-templates/header/global_alert.html

```diff
@@ -0,0 +1,12 @@
+{# Get the current global alert #}
+{%- set global_alert = global_alert() -%}
```

Comment:
What does this do? Cache the global alert so it's only calculated once? Is calculating it expensive?

Reply:
It's not super expensive, but it's used ~5 times in this template, and every time `global_alert()` is called it clones a json value, but it's just not necessary to